.github/workflows/bench.yml

```diff
@@ -11,9 +11,10 @@ jobs:
       - uses: actions/checkout@v3
       - uses: actions/setup-go@v3
         with:
-          go-version: "1.19"
+          go-version: '~1.19.12'
```

Comment:
should we use `>=1.19.12`? What does `~` mean here?

Reply:
It would allow `1.20.x` as well. `~` means `>= 1.9.12, < 1.20.0`

Reply:
Copied this from avalanchego workflows btw.
https://github.com/ava-labs/avalanchego/blob/master/.github/workflows/build-and-test.yml#L23-L24